pep-8002.rst

```diff
@@ -5,6 +5,7 @@ Author: Barry Warsaw <barry@python.org>, Łukasz Langa <lukasz@python.org>,
         Carol Willing <willingc@gmail.com>
 Status: Active
```

Comment:
```suggestion
Status: Final
```

Presumably, this PEP is no longer being actively updated now that the selection process has concluded, so it should presumably be marked Final as well.